distributed-calculator/react-calculator/server.js

```diff
@@ -14,41 +18,92 @@ const daprUrl = `http://localhost:${daprPort}/v1.0/invoke`;
 const stateStoreName = `statestore`;
 const stateUrl = `http://localhost:${daprPort}/v1.0/state/${stateStoreName}`;
 
+const debugMode = process.env.DEBUG_MODE || false; // Emit debug level logs for State management
```

Comment:
Seeing we are working towards this in the JS SDK it might be interesting to use `??` instead of `||`